### ISSUE TITLE
Unable to load web page using webview2 on windows server 2016

### ISSUE THREAD

**Issue by LonelyTiger** [MicrosoftEdge/WebView2Feedback#890]
I have created the sample  WPF application with .net framework 4.6.2 to test webview2.  When I ran the sample application on windows server 2016  i am getting "RenderProcessExited" in the "process Failed" event.  With the similar setup I can able to load the web page works on Windows 10 (OS). 

Version
SDK: 1.0.705.50 
Runtime: Dev 88.0.705.53
Framework: WPF
OS: Windows server 2016

**Comment by champnic** [comment]
Does the Microsoft Edge browser work fine on that same machine?

**Comment by champnic** [comment]
Closing for inactivity. If you are still running into this issue please let me know and I can reopen. Thanks!

**Comment by jdmc1977** [comment]
I had the same error message on both of my 2016 servers, and it was related to remote desktop software similar to Citrix which I will look for some registry changes as suggested for Citrix.  I also got the blank screen only when loading the sample files on both of the servers, and the resolution was to give the msedgewebview2.exe Windows 8 compatibility for all users.  This was the same fix I had to get Edge Chromium working on both of the 2016 servers.  This applied to webview2 V.89.0.774.54

SDK:  1.0.774.44
Runtime: 89.0.774.54
Framework: Windows Forms 4.6.2
OS: Windows Server 2016

![msedgewebview2_server2016](https://user-images.githubusercontent.com/80908389/111692961-6c1db680-8806-11eb-88dd-8414035602ec.png)
